core/input/input.cpp

```diff
@@ -35,6 +35,8 @@
 #include "core/input/input_map.h"
 #include "core/os/os.h"
 
+#include "editor/editor_settings.h"
```

Comment:
Can't access this or the other code without `TOOLS_ENABLED`

Reply:
It' used for editor, I will update the code, thanks!

Reply:
Only available in editor builds so always needed that check unless the file is only editor code itself

Reply:
Thanks! So I guess the `TOOLS_ENABLED` is pretty much like Unity's editor macro.

Reply:
Probably, haven't used Unity